src/cascadia/TerminalSettingsEditor/MainPage.xaml

```diff
@@ -170,18 +170,21 @@
                 <RowDefinition Height="*" />
                 <RowDefinition Height="Auto" />
             </Grid.RowDefinitions>
-            <Frame x:Name="contentFrame"
-                   Grid.Row="0">
-                <Frame.ContentTransitions>
-                    <TransitionCollection>
-                        <NavigationThemeTransition>
-                            <NavigationThemeTransition.DefaultNavigationTransitionInfo>
-                                <DrillInNavigationTransitionInfo />
-                            </NavigationThemeTransition.DefaultNavigationTransitionInfo>
-                        </NavigationThemeTransition>
-                    </TransitionCollection>
-                </Frame.ContentTransitions>
-            </Frame>
+            <ScrollViewer x:Name="scrollBar"
+                          Grid.Row="0">

```

Comment:
Yes, you are right, this would result in nested `<ScrollViewer>`s. However, I do not see how I could control the `<ScrollViewer>` within a `<Frame>`. In addition, since we are `_Navigate()`ing after clicking save in order for the settings to refresh, it is as if we are navigating to a new page, resetting the scroll to the top. This behavior is the same for pages with and without a `<ScrollViewer>`, as some use the built-in scroll of elements such as `<ListView>`.

I think the best way to do this would be to instead of calling `_Navigate()`, to refresh the values of the settings themselves, but I am not entirely sure how to go about that as I'm still getting acquainted with the codebase. Thanks!